exercises/react-vite/02-setting-up-your-environment/solution/src/App.tsx

```diff
@@ -2,9 +2,7 @@ import './App.css'
 
 function App() {
   return (
-    <>
-      <h1>Place My Order App: Coming Soon!</h1>
-    </>
+    <h1>Place My Order App: Coming Soon!</h1>
```

Comment:
It seemed simpler to just replace the entire return statement rather than having a fragment, but if this interferes with how the next module about JSX is designed I can undo this.